report_templates/host_compare_content_hosts_packages.erb

```diff
@@ -0,0 +1,70 @@
+<%#
+name: Host - compare content hosts packages
+snippet: false
+template_inputs:
+- name: Host 1
+  required: true
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+- name: Host 2
+  required: true
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+model: ReportTemplate
+-%>
+<%-  general_lst = [] -%>
+<%-  stage_lst = [] -%>
+<%-  final_lst = [] -%>
+<%-  arch_lst = [] -%>
+<%-  host1 = Host.find_by_name(input("Host 1")) -%>
+<%-  host2 = Host.find_by_name(input("Host 2")) -%>
+<%-  set1 = host1.installed_packages.group_by { |p| p.name } -%>
+<%-  set2 = host2.installed_packages.group_by { |p| p.name } -%>
+<%-  (set1.keys | set2.keys).uniq.each do |package_name| -%>
+<%-    data = { 'Package' => package_name } -%>
+<%-    data[host1.fqdn] = set1[package_name].try(:first).try(:nvra) || '-' -%>
```

Comment:
Bit of an edge case, but the kernel usually has multiple versions installed. Should it attempt to handle this or ignore that edge case?

Reply:
good catch, given we can't tell which kernel is running, I'm not sure it's worth trying to list all versions here, we could potentially order them to get the newest at least, but it can be more misleading

I always thought only one package can actually be installed but kernels remain installed until explicitly purged, but I'm probably mixing this with debian world, where packages contain kernel version in their name and "kernel" package just enforce some version to be installed